languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/en-US/grammar.xml

```diff
@@ -303,6 +303,144 @@ USA
             <example correction="2011,">The meeting scheduled for July 10, <marker>2011</marker> was canceled.</example>
             <example>My birthday is October 8, <marker>1983,</marker> but I usually do not celebrate.</example>
         </rule>
+        <rulegroup id="US_ONE_ENTITY" name="the United States [are] -> the United States [is] 245 years old" default="temp_off">
+            <rule>
+                <antipattern>
+                    <token>States</token>
+                    <token chunk_re="[IE]-NP-plural">
+                        <exception>need</exception>
+                    </token>
+                </antipattern>
+                <antipattern>
+                    <token postag="IN|RN|TO|V.*" postag_regexp="yes" />
+                    <token min="0">that</token>
+                    <token>the</token>
+                    <token min="0" postag="J.*" postag_regexp="yes" />
+                    <token>United</token>
+                    <token>States</token>
+                </antipattern>
+                <antipattern>
+                    <token postag="NNP" />
+                    <token min="0">,</token>
+                    <token>and</token>
+                    <token min="0">the</token>
+                    <token min="0" postag="J.*" postag_regexp="yes" />
+                    <token>United</token>
+                    <token>States</token>
+                </antipattern>
+                <antipattern>
+                    <token postag="MD|WRB" postag_regexp="yes" />
+                    <token min="0" postag="V.*" postag_regexp="yes" />
+                    <token min="0" inflected="yes">not</token>
+                    <token>the</token>
+                    <token>United</token>
+                    <token>States</token>
+                </antipattern>
+                <antipattern>
+                    <token postag="NNS" />
+                    <token>and</token>
+                    <token>the</token>
+                    <token>United</token>
+                    <token>states</token>
+                </antipattern>
+                <antipattern>
+                    <token />
+                    <token postag="," />
+                    <token>and</token>
+                    <token>the</token>
+                    <token>United</token>
+                    <token>states</token>
+                </antipattern>
+                <pattern>
+                    <token>the</token>
+                    <token>United</token>
+                    <token>States</token>
+                    <marker>
+                        <token postag="VB|VBP" postag_regexp="yes">
```

Comment:
I'm not sure about `VB` here, because of "to be". The VB of "be" is "be", the VBP is am|are. I am thinking of cases like "lest the United States be forgotten".

Have you AP'ed the case of the conjunctive mood? "It is imperative that the United States take a firm stance"

Reply:
VB was there solely because "have" is tagged as VB only (not VBP), but I've removed it.

AP for conjunctive mood is the first pattern at the top.

Made a separate rule to deal with "have."